.ci/resources/users.htpasswd

```diff
@@ -1 +1 @@
-user:{SHA}Et6pb+wgWTVmq3VpLJlJWWgzrck=
+che-user:{SHA}Et6pb+wgWTVmq3VpLJlJWWgzrck=
```

Comment:
Is it possible to include a comment here showing what the password is? I think .htpasswd files support `#` for line comments.